emotions.gemspec

```diff
@@ -24,4 +24,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '~> 1.3'
   spec.add_development_dependency 'rspec', '~> 3.1'
   spec.add_development_dependency 'sqlite3', '~> 1.3'
+  spec.add_development_dependency 'phare', '~> 0.7'
+  spec.add_development_dependency 'rubocop', '0.27'
```

Comment:
Is there a reason we’re using this frozen version instead of the latest `0.31` version?